src/supervisor.c

```diff
@@ -137,7 +137,9 @@ void _svNextByte() {
 
 // standaloneModeEnable - enable standalone operation
 void standaloneModeEnable() {
-	svInitFlags = 0x1;

```

Comment:
This didn't work because svInit runs before initIO, so the actual TX SPI buffer never gets updated.